### ISSUE TITLE
Can't graph unified gc log w/o uptime

### ISSUE THREAD

**Issue by pburka** [chewiebug/GCViewer#223]
In Java 11, if you collect GC logs with time, but not uptime (e.g. `-Xlog:gc*=info:gclog:time,tags -XX:+UseParallelGC`), GCViewer seems to parse the logs successfully, but it can't graph them. All of the lines are bunched up against the left hand side of the graph, and throughput calculation is completely wrong (i.e. a large negative number).

`[2019-07-18T18:29:11.553+0000][gc] Using Parallel
[2019-07-18T18:29:11.553+0000][gc,heap,coops] Heap address: 0x000000048f400000, size: 14092 MB, Compressed Oops mode: Zero based, Oop shift amount: 3
[2019-07-18T18:29:12.220+0000][gc,start     ] GC(0) Pause Young (Allocation Failure)
[2019-07-18T18:29:12.289+0000][gc,heap      ] GC(0) PSYoungGen: 196401K->3104K(263680K)
[2019-07-18T18:29:12.289+0000][gc,heap      ] GC(0) ParOldGen: 0K->68341K(602112K)
[2019-07-18T18:29:12.289+0000][gc,metaspace ] GC(0) Metaspace: 14973K->14973K(1062912K)
[2019-07-18T18:29:12.289+0000][gc           ] GC(0) Pause Young (Allocation Failure) 191M->69M(845M) 69.243ms
[2019-07-18T18:29:12.289+0000][gc,cpu       ] GC(0) User=0.33s Sys=0.08s Real=0.07s
[2019-07-18T18:29:12.341+0000][gc,start     ] GC(1) Pause Young (Allocation Failure)
[2019-07-18T18:29:12.391+0000][gc,heap      ] GC(1) PSYoungGen: 223402K->3040K(489984K)
...
`

**Comment by chewiebug** [comment]
Yes - as documented in the [readme](https://github.com/chewiebug/GCViewer), uptime currently is required. for GCViewer to work

Minimum tags required: -Xlog:gc=info:file="path-to-file":tags,uptime,level

I might change this in the future.

**Comment by mehturt** [comment]
Would this be addressed by https://github.com/chewiebug/GCViewer/pull/269 ?

**Comment by awesomefly** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by chewiebug** [comment]
Probably yes!
